go.mod

```diff
@@ -59,6 +59,8 @@ require (
 	gotest.tools/v3 v3.5.1
 )
 
+replace github.com/docker/cli v26.1.3+incompatible => github.com/docker/cli v26.1.1-0.20240607143903-7ae9f2738c6b+incompatible
```

Comment:
Not sure why, but `go mod tidy` kept reverting my change when trying to do that in the `require` block above 🤔 @thaJeztah @glours @ndeloof Any ideas?

Reply:
Because 26.1.1 is lower than 26.1.3. Thank Go for inventing pseudo versions and not understanding release branches

Reply:
So we'll need to make a rc1 tag to make it possible to update github.com/docker/cli here?